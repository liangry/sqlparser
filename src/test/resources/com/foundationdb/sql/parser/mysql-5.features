+MYSQL_COLUMN_AS_FUNCS
