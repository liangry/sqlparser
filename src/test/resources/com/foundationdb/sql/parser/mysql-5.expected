com.foundationdb.sql.parser.CursorNode@56f6fc62
name: null
updateMode: UNSPECIFIED
statementType: SELECT
resultSet: 	
	com.foundationdb.sql.parser.SelectNode@7133b288
	isDistinct: false
	resultColumns: 		
		com.foundationdb.sql.parser.ResultColumnList@790e38c7

		[0]:		
		com.foundationdb.sql.parser.ResultColumn@14b1cee8
		exposedName: null
		name: null
		tableName: null
		isDefaultColumn: false
		type: null
		expression: 			
			com.foundationdb.sql.parser.SpecialFunctionNode@2586ebf5
			sqlName: null
			type: null
	fromList: 		
		com.foundationdb.sql.parser.FromList@3ced3512
